index.html

```diff
@@ -1336,6 +1336,15 @@ <h5>
           <div class="note">
             Other specifications may expand on this definition.
           </div>
+          <div class="note">
+            <p>
+              <i>This note is not normative.</i>
+            </p>
+            When an <code>"insertFromPaste"</code> <code>beforeinput</code>
+            event is dispatched, it should be preceded by a
+            <a href="https://www.w3.org/TR/clipboard-apis/#clipboard-event-paste">paste</a>
```

Comment:
Probably need to fix clipboard api spec to export `paste` definition.

Reply:
@travisleithead @garykac Any change we can get the paste definition from the clipboard apis exported?